rooms.js

```diff
@@ -933,16 +933,34 @@ class BattleRoom extends Room {
 	}
 	requestKickInactive(user, force) {
 		if (this.resetTimer) {
-			if (user) this.sendUser(user, '|inactive|The inactivity timer is already counting down.');
-			return false;
+			let timerSetByUser = user && (this.resetUser === user.userid || (user in this.game.players && this.resetUser === '+'));
```

Comment:
Yeah, I used `'+'` to denote that both players want the timer on, in the same way that `'~'` denotes that an admin turned it on. I had also considered using `'%'` to mean that the timer was forced on by staff, but wasn't sure if that was a good idea.